config/kafka-monitor.properties

```diff
@@ -96,42 +96,52 @@
   }
 
 
-#  Example statsd-service to report metrics
-#  "statsd-service": {
-#      "class.name": "com.linkedin.kmf.services.StatsdMetricsReporterService",
-#      "report.statsd.host": "localhost",
-#      "report.statsd.port": "8125",
-#      "report.statsd.prefix": "kafka-monitor",
-#      "report.interval.sec": 1,
-#      "report.metrics.list": [
-#      "kmf:type=kafka-monitor:offline-runnable-count",
-#      "kmf.services:type=produce-service,name=*:produce-availability-avg",
-#      "kmf.services:type=consume-service,name=*:consume-availability-avg"
-#     ]
-#  }
-
-
-#  Example kafka-service to report metrics
-#  "reporter-kafka-service": {
-#    "class.name": "com.linkedin.kmf.services.KafkaMetricsReporterService",
-#    "report.interval.sec": 3,
-#    "zookeeper.connect": "localhost:2181",
-#    "bootstrap.servers": "localhost:9092",
-#    "topic": "kafka-monitor-topic-metrics",
-#    "report.kafka.topic.replication.factor": 1,
-#    "report.metrics.list": [
-#      "kmf.services:type=produce-service,name=*:produce-availability-avg",
-#      "kmf.services:type=consume-service,name=*:consume-availability-avg",
-#      "kmf.services:type=produce-service,name=*:records-produced-total",
-#      "kmf.services:type=consume-service,name=*:records-consumed-total",
-#      "kmf.services:type=consume-service,name=*:records-lost-total",
-#      "kmf.services:type=consume-service,name=*:records-duplicated-total",
-#      "kmf.services:type=consume-service,name=*:records-delay-ms-avg",
-#      "kmf.services:type=produce-service,name=*:records-produced-rate",
-#      "kmf.services:type=produce-service,name=*:produce-error-rate",
-#      "kmf.services:type=consume-service,name=*:consume-error-rate"
-#    ]
-#  }
+  #  Example statsd-service to report metrics
```

Comment:
Looks like you have an extra space before the `#`; remove so those don't appear as they've changed